api/src/main/java/org/regenstrief/linkage/io/OpenMRSReader.java

```diff
@@ -18,13 +18,16 @@
 import org.openmrs.Patient;
 import org.openmrs.PatientIdentifierType;
 import org.openmrs.PersonAttributeType;
+import org.openmrs.api.context.Context;
 import org.openmrs.module.patientmatching.HibernateConnection;
 import org.openmrs.module.patientmatching.LinkDBConnections;
 import org.regenstrief.linkage.Record;
 
 public class OpenMRSReader implements DataSourceReader {
 
-    private final int PAGING_SIZE = 10000;
+    private final int PAGING_SIZE = Context.getAdministrationService().getGlobalPropertyValue("patientmatching.scratchPageSize", Integer.valueOf(10000)).intValue();
```

Comment:
To avoid a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants), use

```java
private final int DEFAULT_PAGING_SIZE = 10000; // number of patients read at a time
```